test/distributed/_composable/fsdp/test_fully_shard_memory.py

```diff
@@ -114,9 +127,11 @@ def _test_fully_shard_training_memory(self, reshard_after_forward: bool):
             expected_mem_mb = (
                 3 * max_unsharded_numel + non_block_numel
             ) * 4 / 1e6 + buffer_mb
-            # 2x sharded parameters/gradients
-            expected_mem_mb += 2 * model_sharded_numel * 4 / 1e6
+            if not use_cpu_offload:
+                # 2x sharded parameters/gradients
+                expected_mem_mb += 2 * model_sharded_numel * 4 / 1e6
```

Comment:
Should we have some episilon constant here?

Reply:
Sorry, to clarify, what would the epsilon be for?

Reply:
Just some constant so it can be changed easily in the future if we need to for precision reasons (instead of a magic number).

Reply:
I think the `1e6` is for converting to MB rather than being an epsilon in the numerical stability sense.